tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/GitParserTests.cs

```diff
@@ -115,20 +115,54 @@ public void ExtractGitDataFromFolder(TestItem testItem)
-            Assert.Equal(testItem.AuthorName, gitInfo.AuthorName);
-            Assert.Equal(testItem.Branch, gitInfo.Branch);
-            Assert.Equal(testItem.Commit, gitInfo.Commit);
-            Assert.Equal(testItem.CommitterDate, gitInfo.CommitterDate.Value.ToString("u"));
-            Assert.Equal(testItem.CommitterEmail, gitInfo.CommitterEmail);
-            Assert.Equal(testItem.CommitterName, gitInfo.CommitterName);
-            Assert.NotNull(gitInfo.Message);
-            Assert.NotNull(gitInfo.PgpSignature);
-            Assert.Equal(testItem.Repository, gitInfo.Repository);
-            Assert.Equal(testItem.SourceRoot, gitInfo.SourceRoot);
+            // Let's try with the git info provider based on manual parsing of the git folder
+            if (!ManualParserGitInfoProvider.Instance.TryGetFrom(testItem.GitFolderPath, out var gitInfo))
+            {
+                var errors = string.Join(Environment.NewLine, gitInfo?.Errors ?? []);
+                throw new Exception($"Error parsing git info from provider: {nameof(ManualParserGitInfoProvider)}{Environment.NewLine}{errors}");
+            }
+
+            AssertGitInfo(testItem, gitInfo);
+            AssertGitInfo(testItem, GitInfo.GetFrom(testItem.GitFolderPath));
+
+            static void AssertGitInfo(TestItem testItem, IGitInfo gitInfo)
+            {
+                Assert.Equal(testItem.AuthorDate, gitInfo.AuthorDate.Value.ToString("u"));
+                Assert.Equal(testItem.AuthorEmail, gitInfo.AuthorEmail);
+                Assert.Equal(testItem.AuthorName, gitInfo.AuthorName);
+                Assert.Equal(testItem.Branch, gitInfo.Branch);
+                Assert.Equal(testItem.Commit, gitInfo.Commit);
+                Assert.Equal(testItem.CommitterDate, gitInfo.CommitterDate.Value.ToString("u"));
+                Assert.Equal(testItem.CommitterEmail, gitInfo.CommitterEmail);
+                Assert.Equal(testItem.CommitterName, gitInfo.CommitterName);
+                Assert.NotNull(gitInfo.Message);
+                Assert.Equal(testItem.Repository, gitInfo.Repository);
+                Assert.Equal(testItem.SourceRoot, gitInfo.SourceRoot);
+            }
+        }
+
+        [SkippableFact]
+        public void GitCommandGitInfoProviderTest()
+        {
+            if (GitCommandGitInfoProvider.Instance.TryGetFrom(Environment.CurrentDirectory, out var gitInfo))
+            {
+                Assert.NotNull(gitInfo.AuthorDate);
+                Assert.NotNull(gitInfo.AuthorEmail);
+                Assert.NotNull(gitInfo.AuthorName);
+                Assert.NotNull(gitInfo.Branch);
+                Assert.NotNull(gitInfo.Commit);
+                Assert.NotNull(gitInfo.CommitterDate);
+                Assert.NotNull(gitInfo.CommitterEmail);
+                Assert.NotNull(gitInfo.CommitterName);
+                Assert.NotNull(gitInfo.Message);
+                Assert.NotNull(gitInfo.Repository);
+                Assert.NotNull(gitInfo.SourceRoot);
```

Comment:
```suggestion
                gitInfo.AuthorDate.Should().NotBeNull();
                gitInfo.AuthorEmail.Should().NotBeNull();
                gitInfo.AuthorName.Should().NotBeNull();
                gitInfo.Branch.Should().NotBeNull();
                gitInfo.Commit.Should().NotBeNull();
                gitInfo.CommitterDate.Should().NotBeNull();
                gitInfo.CommitterEmail.Should().NotBeNull();
                gitInfo.CommitterName.Should().NotBeNull();
                gitInfo.Message.Should().NotBeNull();
                gitInfo.Repository.Should().NotBeNull();
                gitInfo.SourceRoot.Should().NotBeNull();
```

Reply:
done in [d7bebae](https://github.com/DataDog/dd-trace-dotnet/pull/6259/commits/d7bebae2cda5caa60fd13518156b61df01d04162)